nilearn/plotting/__init__.py

```diff
@@ -93,35 +93,35 @@ def _set_mpl_backend():
 )
 
 __all__ = [
```

Comment:
Probably want to make sure the __all__ are sorted alphabetically to make easier to cross check things